README.md

```
@@ -1,755 +1,300 @@
-- [Testing](#-testing)
-- [Deployment](#-deployment)
-- [Security](#-security)
-- [Contributing](#-contributing)
-- [License](#-license)
-
----
-
-## 🏗 Architecture
-
-HIVE-MIND follows a strict **layered architecture** with one-way dependency flow, governed by a declarative blueprint system and a cognitive layer:
+## <img src="https://cdn.jsdelivr.net/gh/withxat/fluentui-emoji-unicode@webp/assets/1f3d7_3d.webp" width="28" height="28" style="vertical-align: middle; margin-right: 8px;" /> Architecture
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)